scalac-scoverage-plugin/src/main/scala/scoverage/report/ScoverageXmlWriter.scala

```diff
@@ -4,7 +4,10 @@ import java.io.File
 
 import scoverage._
 
-import scala.xml.{Node, PrettyPrinter}
+import javax.xml.stream.XMLOutputFactory
+import javax.xml.stream.XMLStreamWriter
+import java.io.FileWriter
+import com.sun.xml.internal.txw2.output.IndentingXMLStreamWriter
```

Comment:
This is an internal class. Can it be returned by some public factory to avoid referencing it directly?

Reply:
https://bugs.openjdk.java.net/browse/JDK-8141405